app/src/pages/inside/common/statusDropdown/statusDropdown.jsx

```diff
@@ -147,12 +147,8 @@ export class StatusDropdown extends Component {
     const { status, withIndicator, disabled, readOnly } = this.props;
     return (
       <div className={cx('status-dropdown')}>
-        {readOnly ? (
-          <TestItemStatus
-            status={status}
-            withIndicator={withIndicator}
```

Comment:
Please also remove the `withIndicator` prop usage in line `136` as well if it is not needed anymore.